### DISCUSSION TITLE
Which variable to adjust number of iterations in Web UI?

### DISCUSSION THREAD

**Discussion by Wonderflex** [hlky/stable-diffusion-webui#1155] [Q&A] [answered]
When running the webui, the slider for number of iterations is capped at 50.  Which file and line would I need to edit to uncap this, or change it to a free text entry?  I searched the webui.py and can't seem to find it.

**Comment by codedealer** [comment, marked as answer]
https://github.com/sd-webui/stable-diffusion-webui/discussions/1096
Take a look here, the `batch_count` is edited in the same file a few lines up

**Reply by Wonderflex** [reply]
Thank you very much.   I searched for this, but obviously not good enough.
